spec/javascript/components/lease_ups/LeaseUpsPage.test.js

```diff
@@ -39,61 +73,25 @@ describe('LeaseUpApplicationsPage', () => {
   sharedHooks.useFakeTimers()
 
   test('Should render LeaseUpTable', () => {
-    const applications = [ buildApplicationPreference(1), buildApplicationPreference(2) ]
     const wrapper = renderer.create(
-      <LeaseUpApplicationsPage listing={listings} applications={applications} />
+      <LeaseUpApplicationsPage listing={listing} />
     )
 
     expect(wrapper.toJSON()).toMatchSnapshot()
   })
 
   test('Should render Mailing_Address when present', () => {
-    const applications = [
-      buildApplicationPreference(1, {
-        'Application': {
-          'Applicant': {
-            'Residence_Address': '',
-            'Mailing_Address': '1316 SUTTER'
-          }
-        }
-      })
-    ]
-
     const wrapper = renderer.create(
-      <LeaseUpApplicationsPage listing={listings} applications={applications} />
+      <LeaseUpApplicationsPage listing={listing} />
     )
 
     expect(wrapper.toJSON()).toMatchSnapshot()
   })
 
   test('Should render order by Prefrence_Order and Preference_Lottery_Rank', () => {
```

Comment:
typo in Preference order here
```suggestion
   test('Should render order by Preference_Order and Preference_Lottery_Rank', () => {
```